health-services/project-factory/src/server/utils/campaignMappingUtils.ts

```
@@ -197,15 +343,21 @@
+
+
     for (const key of Object.keys(boundaryWithProject)) {
         if (boundaryWithProject[key]) {
             const resources: any[] = [];
-            const pvarIds = getPvarIds(messageObject);
-            if (pvarIds && Array.isArray(pvarIds) && pvarIds.length > 0) {
-                resources.push({
-                    type: "resource",
-                    resourceIds: pvarIds
-                })
+            if (messageObject?.Campaign?.newlyCreatedBoundaryProjectMap?.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)